PAYMENTS.md

```diff
@@ -0,0 +1,27 @@
+# Integrate your Stripe account to accept your user's payments
+All the information that you need to setup your Stripe account into OSEM.
+If you have any problems with installing don't hesitate to [contact us](https://github.com/openSUSE/osem#contact)
+
+## Configure Stripe into the application
+To configure Stripe into your application all you need to do is add the private and publishable keys into the Rails environment.
+
+You can add the your keys in `dotenv.example` and in `config/secrets.yml` into these variables:
```

Comment:
If you document something tell people what to do, do not provide them a menu of options :-)